index.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   rules: {
+    'no-stubbing-in-nested-before-each': require('./rules/noStubbingInNestedBeforeEach'),
```

Comment:
Maybe `noStubbingInNestedBeforeEach.js` -> `no-stubbing-in-nested-before-each.js`?
https://github.com/eslint/eslint/tree/master/lib/rules

Reply:
I was actually thinking of that, great shout tho 👍